src/main/java/duke/UserInterface.java

```diff
@@ -0,0 +1,232 @@
+package duke;
+
+import duke.command.Command;
+
```

Comment:
Consider separating UserInterface class into Parser and UI to for further abstraction. Currently it is fine regardless

Reply:
Thank you. Might add the Parser class if time permits.